src/project.js

```diff
@@ -64,6 +64,8 @@ function projectionReducer(location, targetAltitude) {
       newLocation.forestType = forestType;
     }
     newLocation.altitudinalZone = nextAltitudinalZone(location.altitudinalZone);
+  } else if (location.altitudinalZone === undefined) {
```

Comment:
Unfortunatly this is a little bit too simple. `altitudinalZone` can be availble but forestEcoregion is still not found in projections. So better check if provided forestEcoregions is found and otherwise do not return location information (just options).

Reply:
Done!